Assets/MixedRealityToolkit/_Core/Devices/OpenVR/GenericOpenVRController.cs

```diff
@@ -19,36 +20,103 @@ public GenericOpenVRController(TrackingState trackingState, Handedness controlle
-
-        #region Base override configuration
-
-        /// <summary>
-        /// Mapping method to expose the Unity Input Manager mapping configuration
-        /// </summary>
-        public override InputManagerAxis[] ControllerAxisMappings => ControllerInputAxisMappingLibrary.GetInputManagerAxes(GetType().FullName);
-
-        /// <summary>
-        /// Mapping method to expose this controllers Unity Input Manager mapping array
-        /// </summary>
-        public virtual string[] VRInputMappings => ControllerInputAxisMappingLibrary.GetInputManagerMappings(GetType().FullName);
-
-        #endregion Base override configuration
+        public static readonly MixedRealityInteractionMapping[] DefaultLeftHandedInteractions =
+        {
+            // Controller Pose
+            new MixedRealityInteractionMapping(0, "Spatial Pointer", AxisType.SixDof, DeviceInputType.SpatialPointer, MixedRealityInputAction.None),
+            // HTC Vive Controller - Left Controller Trigger (7)
+            // Oculus Touch Controller - Axis1D.PrimaryIndexTrigger
+            // Valve Knuckles Controller - Left Controller Trigger
+            new MixedRealityInteractionMapping(1, "Trigger Position", AxisType.SingleAxis, DeviceInputType.Trigger, MixedRealityInputAction.None, KeyCode.None, ControllerMappingLibrary.MIXEDREALITY_OPENVR_AXIS9),
+            // HTC Vive Controller - Left Controller Trigger (7)
+            // Oculus Touch Controller - Axis1D.PrimaryIndexTrigger
+            // Valve Knuckles Controller - Left Controller Trigger
+            new MixedRealityInteractionMapping(2, "Trigger Touch", AxisType.Digital, DeviceInputType.TriggerTouch, MixedRealityInputAction.None, KeyCode.JoystickButton15),
+            // HTC Vive Controller - Left Controller Trigger (7)
+            // Oculus Touch Controller - Axis1D.PrimaryIndexTrigger
+            // Valve Knuckles Controller - Left Controller Trigger
+            new MixedRealityInteractionMapping(4, "Trigger Press", AxisType.Digital, DeviceInputType.TriggerPress, MixedRealityInputAction.None, KeyCode.None, ControllerMappingLibrary.MIXEDREALITY_OPENVR_AXIS9),
+            // HTC Vive Controller - Left Controller Grip Button (8)
+            // Oculus Touch Controller - Axis1D.PrimaryHandTrigger
+            // Valve Knuckles Controller - Left Controller Grip Average
+            new MixedRealityInteractionMapping(5, "Grip", AxisType.SingleAxis, DeviceInputType.Trigger, MixedRealityInputAction.None, KeyCode.None, ControllerMappingLibrary.MIXEDREALITY_OPENVR_AXIS11),
+            // HTC Vive Controller - Left Controller Trackpad (2)
+            // Oculus Touch Controller - Axis2D.PrimaryThumbstick
+            // Valve Knuckles Controller - Left Controller Trackpad
+            new MixedRealityInteractionMapping(10, "Trackpad-Thumbstick Position", AxisType.DualAxis, DeviceInputType.Touchpad, MixedRealityInputAction.None),
```

Comment:
What do these id numbers refer to, and why do they jump from 5 to 10?

Reply:
Lol, Ask Unity 🗡 
https://docs.unity3d.com/Manual/OpenVRControllers.html

Reply:
Again the `DeviceInputType.Touchpad != DeviceInputType.Thumbstick`

Reply:
So they're the...Unity Axis ID? The values here don't seem to match the values there. 10 is `Trackpad-Thumbstick Position` here but Right Controller Trigger there.